client/src/components/index.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/', require('../controllers/home').get);
```

Comment:
Requires should be done on the top level for readability

Reply:
this whole file was added also somehow so this whole file will be removed ! :)
Thanks !